doc/metals.txt

```diff
@@ -990,9 +990,33 @@ The following configuration values are available to be passed into
     node_command_mapping = "r",
     collapsed_sign = "▸",
     expanded_sign = "▾",
+    icons = {
+      enabled = false,
+      symbols = {
+        val = "",
+        field = "",
+        method = "ﬦ",
+        constructor = "",
+        object = "",
+        class = "ﭰ",
+        trait = "",
+        interface = "",
+        package = "",
+        package_object = "",
```

Comment:
I see that the name of an icon is of type `Kind` which is defined here https://github.com/scalameta/scalameta/blob/main/semanticdb/semanticdb/semanticdb.proto#L246-L264

But I didn't see any of `package_object`, `macro`, `type`, `parameter`, `self_parameter`, `type_parameter` or `local` icons in the tvp. So maybe it is a good idea to delete them from here.

Reply:
Yea, actually you don't need quite a few of these. There is a minimal set that are used. You can actually see them all [here](https://github.com/scalameta/metals/blob/main/metals/src/main/scala/scala/meta/internal/tvp/ClasspathTreeView.scala#L98-L111). So you should be able to simplify this to those.

Reply:
awesome, thanks!